Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[workspace]
+resolver = "2"
+members = [
+    "kvm-bindings",
+    "kvm-ioctls",
+]
+
+[workspace.dependencies]
```

Comment:
Why does this need to be defined as a workspace dependency?

Reply:
IMHO keeping common dependencies here could help us better manage the monorepo, and send the message that we want these dependencies stay in sync :)

Reply:
Looking at the documentation of workspace, it seems like you can't define the dependencies as optional: https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-dependencies-table. But we do need it to be optional for kvm-bindings. How does that work?


In any case, I would propose this change to be done in a separate PR because it is not strictly required for this merge, and we should discuss it separately.

Reply:
> Looking at the documentation of workspace, it seems like you can't define the dependencies as optional: https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-dependencies-table. But we do need it to be optional for kvm-bindings. How does that work?
> 
> In any case, I would propose this change to be done in a separate PR because it is not strictly required for this merge, and we should discuss it separately.

Good point, I will drop it then